### PR TITLE
Prevent mp rest client tck from running on slow windows machines, avoiding bogus build-break defects

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

**Comment by jim-krueger** [comment]
#libby #build

**Comment by jim-krueger** [comment]
#libby #build

**Comment by jim-krueger** [comment]
#libby #build

File: dev/com.ibm.ws.microprofile.rest.client_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/ProduceConsumeTest.java [hunk 1]
```diff
@@ -35,19 +37,34 @@
...
-                                                             MicroProfileActions.MP33, // 1.4
-                                                             MicroProfileActions.MP40, // 2.0
-                                                             MicroProfileActions.MP50, // 3.0
-                                                             MicroProfileActions.MP60);// 3.0+EE10
+    public static RepeatTests r;
+    static {
+        if (!(isWindows) || FATRunner.FAT_TEST_LOCALRUN) {
+            r = MicroProfileActions.repeat(SERVER_NAME, 
+                                           MicroProfileActions.MP14, // 1.1 + EE7
+                                           MicroProfileActions.MP20, //mpRestClient-1.1
```
Comment (jhanders34) [inline dev/com.ibm.ws.microprofile.rest.client_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/ProduceConsumeTest.java]:
```suggestion
                                           MicroProfileActions.MP20, // 1.1 + EE8
```

**Review by jhanders34 — DISMISSED** [review]

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Pull request merged**